web/src/lib/components/asset-viewer/asset-viewer.svelte

```diff
@@ -395,10 +395,10 @@
 
       notificationController.show({
         type: NotificationType.Info,
-        message: asset.isFavorite ? `Added to favorites` : `Removed from favorites`,
+        message: $t(asset.isFavorite ? 'added_to_favorites' : 'removed_from_favorites'),
```

Comment:
The automated test for missing translation doesn't work for dynamic translation keys, so I think we should avoid using them.

```suggestion
        message: asset.isFavorite ? $t('added_to_favorites') : $t('removed_from_favorites'),
```